### ISSUE TITLE
Cannot download HDRI from HDRI haven through Gaffer built in function

### ISSUE THREAD

**Issue by ParanoidNemo** [gregzaal/Gaffer#141]
Gaffer bought from blendermarket.com

**Describe the bug**
Cannot download HDRI from HDRI Haven using Gaffer in build function

**To Reproduce**
1. Go to Gaffer option menù
2. Click on update HDRI list
3. ```
Python: Traceback (most recent call last):
  File "/home/nemo/.config/blender/3.0/scripts/addons/Gaffer/operators.py", line 2018, in invoke
    return context.window_manager.invoke_props_dialog(self, width=300 * fn.dpifac())
TypeError: WindowManager.invoke_props_dialog(): error with keyword argument "width" -  Function.width expected an int type, not float

location: <unknown location>:-1
```

**Expected behavior**
Start downloading HDRI

**Desktop (please complete the following information):**
 - OS: EndeavourOS (arch based)
 - Blender 3.0.1
 - Gaffer 3.1.9

**Comment by gregzaal** [comment]
Thanks for the report, will fix in the next release.

As a workaround, set your DPI scaling in the user preferences to 1.0 and then download the HDRIs.

**Comment by gregzaal** [comment]
Fixed in 5e32190f
